docs/en/compute/AWS-EMR.md

```diff
@@ -41,7 +41,7 @@ To begin with, [download an Alluxio release](https://www.alluxio.io/download) an
 ```console
 $ aws emr create-default-roles
 ```
-2. Download `alluxio-emr.sh` from [our github](https://github.com/Alluxio/alluxio/tree/master/integration/emr/) and
+2. Download `alluxio-emr.sh` from the [alluxio public S3 bucket](https://alluxio-public.s3.amazonaws.com/emr/2.0.1/alluxio-emr.sh) and
```

Comment:
what about `integration/emr/alluxio-emr.json`, where is this file living now? @LuQQiu

Reply:
User's don't need to download from the public bucket unless they want to actually make changes to the bootstrap. They can provide our S3 URI directly and run 'aws emr create-cluster' off of that.